src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,31 @@
+.catalog {
+  display:grid;
+  grid-template-columns: $card-width;
+  justify-content: center;
+  justify-items: center;
+  gap: 48px;
+
+  @media (min-width:488px) and (max-width:768px) {
+    grid-template-columns: repeat(2, $card-width)
+  }
+  @media (min-width:768px) and (max-width:1024px) {
+    grid-template-columns: repeat(3, $card-width)
+  }
+  @media (min-width:1024px){
+    grid-template-columns: repeat(4, $card-width)
+  }
```

Comment:
You use the same template for query every time, try to rewrite it using each loop, so you don`t need to duplicate your code.